app/models/feedback.rb

```diff
@@ -0,0 +1,3 @@
+class Feedback < ApplicationRecord
```

Comment:
I think this class name is not specific enough. Maybe AssignmentFeedback or AssignmentSuggestion ?

That would help reduce confusion with the unrelated 'FeedbackFormResponse', which is a more general set of "feedback" records from many different parts of the application.